content-scripts/inject/run-userscript.js

```diff
@@ -9,6 +9,7 @@ export default async function runAddonUserscripts({ addonId, scripts, enabledLat
     const loadUserscript = async () => {
       await scratchAddons.l10n.loadByAddonId(addonId);
       const module = await import(scriptUrl);
+      await addonObj.tab.scratchClassReady();
```

Comment:
In the future we can probably speed this up a bit with `Promise.all()`
This change does not make it slower though. But the l10n and module import could happen in parallel.

Reply:
Is there a reason for not making this improvement now?

Reply:
Well, technically the bug fix for this regression is also a speed up (just noticed) because the order of the await calls has been changed, so it can be done now.

Reply:
Done